webviz_subsurface/_abbreviations/reservoir_simulation.py

```diff
@@ -30,47 +30,62 @@ def simulation_vector_base(vector: str) -> str:
     """Returns base name of simulation vector
     E.g. WOPR for WOPR:OP_1 and ROIP for ROIP_REG:1
     """
-    return vector.split(":", 1)[0].split("_", 1)[0]
+    return vector.split(":", 1)[0].split("_")[0]
 
 
 def simulation_vector_description(vector: str) -> str:
     """Returns a more human friendly description of the simulation vector if possible,
      otherwise returns the input as is.
     """
     [vector_name, node] = vector.split(":", 1) if ":" in vector else [vector, None]
+    fip = None
+    comp = None
     if len(vector_name) == 8:
-        # Region vectors for other FIP regions than FIPNUM are written on a special form:
-        # 8 signs, with the last 3 defining the region.
-        # E.g.: For an array "FIPREG": ROIP is ROIP_REG, RPR is RPR__REG and ROIPL is ROIPLREG
-        # Underscores _ are always used to fill
-        [vector_base_name, fip] = [vector_name[0:5].rstrip("_"), vector_name[5:]]
-        try:
-            if SIMULATION_VECTOR_TERMINOLOGY[vector_base_name]["type"] == "region":
-                vector_name = vector_base_name
-            else:
-                fip = None
-        except KeyError:
-            fip = None
-    else:
-        fip = None
+        if vector_name[0] == "R":
+            # Region vectors for other FIP regions than FIPNUM are written on a special form:
+            # 8 signs, with the last 3 defining the region.
+            # E.g.: For an array "FIPREG": ROIP is ROIP_REG, RPR is RPR__REG and ROIPL is ROIPLREG
+            # Underscores _ are always used to fill
+            [vector_base_name, fip] = [vector_name[0:5].rstrip("_"), vector_name[5:]]
+            try:
+                if SIMULATION_VECTOR_TERMINOLOGY[vector_base_name]["type"] == "region":
+                    vector_name = vector_base_name
+            except KeyError:
+                pass
```

Comment:
`type` should always be defined I guess, if `vector_base_name` exists, so catching `KeyError` might be slightly loose? Maybe
```suggestion
            if vector_base_name in SIMULATION_VECTOR_TERMINOLOGY and SIMULATION_VECTOR_TERMINOLOGY[vector_base_name]["type"] == "region":
                    vector_name = vector_base_name
```

Reply:
Would say that these two options are equally loose imo, but can change it. Should do it for `type == completion` too in that case.

Reply:
The first will fail the `try` clause both if `vector_base_name` is not defined, and if `type` is (wrongly?) missing from the definition (if `vector_base_name` exists). The latter will not silently fail if the definition is (wrongly?) missing a `type` key for some vector that exists.

Yes, if we change we should change the other branches as well.

Reply:
I agree that from our side it is best that it fails, but for a user it might be better that it is quite ;) 
But of course, if we later include the option of user defined extensions, then an error might be a good thing.
I'll change it.